kahuna/public/stylesheets/main.css

```diff
@@ -308,9 +308,24 @@ a:focus {
     display: inline-block;
 }
 
-/* ==========================================================================
-   Generics
-   ========================================================================== */
+.side-padded {
+    padding: 0 10px;
+}
+
+.clickable:hover {
+    background-color: #666666;
+    color: white;
+}
+
+.inner-clickable {
+    height: 100%;
+    display: inline-block;
+}
+
+.button-right-side {
+    padding-left: 5px;
+    margin-left: -4px
```

Comment:
"clickable" styles the element to suggest to a user they could click on it. 

"inner-clickable" prevents dead space, which cannot be clicked, on the sides of an element.

"button-right-side" allows you to add a dropdown menu button to an element. 

> I don't mind, I just wondered if clickable was the right word - but I suppose something can look clickable. :+1:

alternative names welcome!